fastestimator/op/tensorop/meta/repeat.py

```diff
@@ -161,10 +234,14 @@ def _tf_cond(self, cnd: List[Tensor], data: Dict[str, Tensor]) -> bool:
         Returns:
             Whether to continue looping.
         """
+        if isinstance(self.repeat, int):
+            # In this case we have 2 Forward calls for tf (one before and one after the while loop
+            # (For accurate Tf while loop functioning))
+            return tf.less(cnd, self.repeat - 1)
```

Comment:
wouldn't this be self.repeat - 2?

Reply:
In line 190 , we start with cnd = tf.Constant(1), thats why 'self.repeat - 1' works